backend/routes/domainOfActivityRoute.js

```diff
@@ -0,0 +1,13 @@
+const router = require('express').Router();
+
+const domainOfActivityService = require('../services/domainOfActivityService');
+
+router.get('/domainOfActivity', async (req, res, next) => {
```

Comment:
As I mentioned in my last comment, this should be moved in employerRoute

Reply:
same as above